README.md

```diff
@@ -50,9 +50,23 @@ Benchmarks:
 * Blender: [scene-BMW](https://www.blender.org/download/demo-files/).
 ---
 
-### Unlimited
+### gWASM
 
-* [Installing and testing](Products/Unlimited/Basic-Usage?id=installing-and-testing)
+?> Any code that can be compiled to WASM, can be distributed on Golem Network. gWASM allows you to run WebAssembly binaries on providers’ machines by turning WASM into a container for server-side parallel computations.
```

Comment:
?> Is this needed?

Reply:
As an explaining intro shortly summarizing what one could do with gWASM I think - yes, it is quite useful